egs/tunisian_msa/s5/local/qcri_buckwalter2utf8.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cut -d " " -f 1 qcri.txt > qcri_words_buckwalter.txt
```

Comment:
this should probably be 
 cat qcri.txt | tail -n +4 | cut -d " " -f 1  > qcri_words_buckwalter.txt
 cat qcri.txt | tail -n +4 | cut -d " " -f 2 > qcri_prons.txt
Otherwise lines like "# Copyright" will be included.